nncf/quantization/algorithms/weight_compression/openvino_backend.py

```diff
@@ -109,6 +110,8 @@ def mean_statistic_collector(
 
     @staticmethod
     def get_activation_port_id(node: NNCFNode, nncf_graph: NNCFGraph) -> int:
+        if node.layer_attributes.input_attributes["transpose"]:
+            raise nncf.UnsupportedModelError("Transposed input is not supported")
```

Comment:
Why did you add runtime error instead of introduce a fix? Is there any plan?

Reply:
This was introduce there - https://github.com/openvinotoolkit/nncf/pull/3002. I did not find any ticket on that. Probably, @ljaljushkin can have it

Reply:
https://github.com/openvinotoolkit/nncf/issues/3230